### PR TITLE
2.0.0 Release documentation

### PR DESCRIPTION
**Pull request by kthompson23**
Open documentation branch for all changes that will go out in the next 2.0.0 release.

This PR documents the new maximum return payload size parameter introduced in #357 .

I removed the default value on each of the method documentation and indicated that the value specified at connection creation would be used. I also fleshed out the explanation on the create connection documentation.

### CODE REVIEW

File: website/docs/02 - Connection.md [hunk 1]
```diff
@@ -27,13 +27,14 @@ Connection.createConnection(mvisUri: string, account: string, options?: CreateCo
...
+| `httpsAgent`           | `https.Agent` |               | An `https.Agent` instance to use with https requests (recommended)                                                                                                                                                                                                                                                                                                                                                                     |
+| `maxReturnPayloadSize` | `number`      | `100_000_000` | The maximum allowed return payload size in bytes. If this size is exceeded a DbServerError will be thrown. Returning large payloads can have a significant impact on performance and is often the result of invalid database records or an improperty configured query; ie forgetting to use pagination. Tune this value to match your datasets and the type of queries you issue. This can also be configured on a per request basis. |
```
Comment (shawnmcknight) [inline website/docs/02 - Connection.md:37]: Typo -- `improperty`

**Review by shawnmcknight — CHANGES_REQUESTED** [review]
Looks good.  Just the one typo noted.

File: website/docs/02 - Connection.md (hunk 1 above)
Comment (kthompson23) [reply website/docs/02 - Connection.md]: updated.

**Comment by kthompson23** [comment]
I fixed the issue. This is ready for another look.

**Review by shawnmcknight — APPROVED** [review]
LGTM!

**Pull request merged**